databases/week3/sqlHW.sql

```diff
@@ -0,0 +1,246 @@
+CREATE DATABASE Homework3DB;
+
+USE Homework3DB;
+
+CREATE TABLE Meal (
+    `id` INT NOT NULL AUTO_INCREMENT,
+    `title` VARCHAR(255),
+    `description` TEXT,
+    `location` VARCHAR(255),
+    `when` DATETIME,
+    `maxreservation` INT,
+    `price` DECIMAL(10, 2),
+    `created_date` DATE,
+    PRIMARY KEY(`id`)
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+
+CREATE TABLE Reservation(
+    `id` INT NOT NULL AUTO_INCREMENT,
+    `number_of_guests` INT,
+    `meal_id` INT,
+    `created_date` DATE,
+    `contact_phone_number` VARCHAR(255),
+    `contact_name` VARCHAR(255),
+    `contact_email` VARCHAR(255),
+    PRIMARY KEY (`id`),
+    FOREIGN KEY (meal_id) REFERENCES `Meal` (`id`)
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+
+CREATE TABLE Review(
+    `id` INT NOT NULL AUTO_INCREMENT,
+    `title` VARCHAR(255),
+    `description` TEXT,
+    `meal_id` INT,
+    `stars` INT,
+    `created_date` DATE,
+    PRIMARY KEY (`id`),
+    FOREIGN KEY (meal_id) REFERENCES `Meal` (`id`)
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+
+INSERT INTO
+    Meal (
+        title,
+        description,
+        location,
+        `when`,
+        maxreservation,
+        price,
+        created_date
+    )
+VALUES
+    (
+        'Borsch',
+        'Ukrainian soup',
+        'Vangvej 12, 7950 Erslev',
+        '2023-03-01 18:00:00',
+        12,
+        55.50,
+        '2023-02-28'
+    );
+
+INSERT INTO
+    Meal (
+        title,
+        description,
+        location,
+        `when`,
```

Comment:
Why did you place when in ``?

Reply:
> Well done🤩 Looks like you did a great job especially in learning MongoDB!

😄Thank you, Veronika :) 
It was a little bit hard to understand relationships and foreign keys in mongo.
Thanks for your review 😊

Reply:
Because WHEN is reserved word is SQL. 
And I was caught again yesterday on it, when wrote queries in NodeJS🤦‍♂️😂

Reply:
I see🤔 that's why I suggest to put all your columns names in ‘‘😊

Reply:
Hmm… Yeah, it could be a solution😌
You, guys, always do that?

Reply:
Definitely I'm doing it now😅 but I think I did it for my similar HM.

Reply:
Okay, then I do this changes and will merge after :)
Thanks again for review 😊